src/pages/Home/Home.tsx

```diff
@@ -9,24 +13,32 @@ export const Home = () => {
         <div className={style.heroContainer}>
           <div className={style.heroSection}>
             <NavBar pageName={"home"} />
+            <div className={style.imgContainer}>
+              <img src={logo} alt="logo" />
+            </div>
             <div className={style.titleInfo}>
               <h1>The Royal Canine</h1>
```

Comment:
The name is missing part of it? 
![image](https://github.com/Kenzie-Tech-Solutions/theRoyalCanine/assets/88916004/1f57c297-aaf9-4dd4-8ffe-97f024da1add)
![image](https://github.com/Kenzie-Tech-Solutions/theRoyalCanine/assets/88916004/181bed40-015a-403a-87b4-5f644f70c6ed)